README.md

```diff
@@ -1,21 +1,25 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+A click handler triggers an event on a click to some target in the browser
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+should put it in the document.ready() part of your js so it only runs after everything on the page has loaded/is ready to be manipulated
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+this means that the variable/method has not been set to a value, which means you can fix it by giving it a value with the equals operator.
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+everything that should be run after the page loads.
```

Comment:
Yes, and more specifically, any click handler that should be enabled right when the page loads. The function definitions themselves don't need to go in the `$(document).ready()`.